es/index.go

```diff
@@ -480,6 +481,7 @@ func (index *BaseIndex) RemoveFromIndexQuery(elasticScope elastic.Query) (map[st
 			if _, ok := removed[scrollResult.ResultType]; !ok {
 				removed[scrollResult.ResultType] = make(map[string]bool)
 			}
+			index.esc.ClearScroll(scrollResult.ScrollId)
```

Comment:
Is this the only place where we have scroll that need to be cleared?

Reply:
Seems to be yes. The only other parts we use scroll in the code is inside GetLattestQueries and GetAllClicks (not in index).